cmd/deploy/deploy.go

```diff
@@ -656,6 +677,37 @@ func (dc *DeployCommand) cleanUp(ctx context.Context) {
 	}
 }
 
+func (dc *DeployCommand) createTempOktetoEnvFile() (afero.File, error) {
+	oktetoEnvFileDir, err := afero.TempDir(dc.Fs, "", "")
+	if err != nil {
+		return nil, err
+	}
+
+	oktetoEnvFile, err := dc.Fs.Create(fmt.Sprintf("%s/.env", oktetoEnvFileDir))
+	if err != nil {
+		return nil, err
+	}
+
+	os.Setenv(constants.OktetoEnvFile, oktetoEnvFile.Name())
```

Comment:
Why do we need to set this env var?

Reply:
to be able to use `$OKTETO_ENV` in the commands that add env to the file, for example : `echo "KEY=value" >> $OKTETO_ENV`